src/rpcrawtransaction.cpp

```diff
@@ -334,12 +334,18 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
             "      ,...\n"
             "    }\n"
 
+            "\n"
+            "    If you want to create OP_RETURN output instead of an address output, entries should be in the following format:\n"
+            "\n"
+            "      \"OP_RETURN\": \"hex\" (string, required) The key is \"OP_RETURN\", the value is hex encoded data\n"
```

Comment:
Key probably should be named "commitment" or "data", not "OP_RETURN" (which is just a technical implementation detail).